package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "marked-extensions",
-  "version": "0.5.0",
+  "version": "1.0.0",
```

Wasn't expecting v1. I think we have more to discuss, clean up, and should make sure this can be actually used. I can also set up automation, so we don't need to commit `dist/`.

I'd export the extension as `Renderer` instead of a function that takes `marked`. That makes this much more usable.

So instead of

```javascript
process(marked, markdown, { language: 'javascript' });
```

```javascript
marked(markdown, { renderer: new Renderer({ language: 'javascript' }) });
```

In our apps, we can override or adjust something when necessary by `class CustomRenderer extends Renderer {}`. This also makes `extensions` block unnecessary because it can be done in `CustomRenderer`.

---

I'll open issues that I think we need to resolve before v1.

---

Yeah I went for v1 because it was technically breaking changes. Since we are going full milestone on this, @ggorlen you might want to chime in with your notes.

---

Minor bumps before v1 are breaking changes, so we don't need to jump to v1 for semver.

---

Going for v1 sounds good, but let's keep the version in this PR, so we can bump the version before actually releasing (possibly by automation).

```suggestion
  "version": "0.5.0",
```

---

@ggorlen Can you open a new issue to discuss? I also have some inputs for the generator that I'll probably open later.